### PR TITLE
Update Kotlin/Native links: the development is moved to Kotlin repo

### PR DESCRIPTION
**Pull request by SvyatoslavScherbina**

### CODE REVIEW

File: docs/topics/apple-framework.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Kotlin/Native provides bi-directional interoperability with Objective-C/Swift.
 Objective-C frameworks and libraries can be used in Kotlin code.
 Kotlin modules can be used in Swift/Objective-C code too.
 Besides that, Kotlin/Native has
-[C Interop](https://github.com/JetBrains/kotlin-native/blob/master/INTEROP.md).
+[C Interop](native-c-interop.md).
```
Comment (SvyatoslavScherbina) [inline docs/topics/apple-framework.md:7]: I'm not sure how to properly use internal links. Please let me know if this is wrong.

File: docs/topics/native/native-faq.md [hunk 2]
```diff
@@ -136,28 +136,10 @@ good practise to have the global state immutable. If for some reason you need a
 object, use the `@konan.ThreadLocal` annotation on the object. Also the `kotlin.native.concurrent.AtomicReference` class could be
 used to store different pointers to frozen objects in a frozen object and automatically update them.
 
-## How can I compile my project against the Kotlin/Native master?
+## How can I compile my project with unreleased versions of Kotlin/Native?
 
-One of the following should be done:
+First, please consider trying [preview versions](eap.md).
```
Comment (SvyatoslavScherbina) [inline docs/topics/native/native-faq.md:141]: I'm not sure how to properly use internal links. Please let me know if this is wrong.

File: docs/topics/apple-framework.md (hunk 1 above)
Comment (p7nov) [reply docs/topics/apple-framework.md:7]: Yep, this is correct.

**Review by p7nov — APPROVED** [review]
👍

**Pull request merged**